setup.py

```diff
@@ -31,7 +31,7 @@
 
 # Get the long desc
 with open("README.md", "r") as fh:
-    LONG_DESCRIPTION = fh.read()
+    LONG_DESCRIPTION = fh.read().replace(os.linesep, '\n')
 
```

Comment:
Do we still need this? Maybe good to keep it in case setuptools goes crazy again :)

Reply:
I actually think this line is useless. When the file is read, it will either always have Unix newlines or Python can handle it (if someone changed the line endings and pushed for some reason...). The error in the issue I linked above happened at WRITE time (while building), which we can’t really control. So I will probably remove this before I merge.

Reply:
@aaronreidsmith Roger that 💯